vizro-ai/docs/pages/user-guides/run-vizro-ai-dashboard.md

```diff
@@ -0,0 +1,175 @@
+# How to run Vizro-AI dashboard
+
+This guide offers insights into different ways of running `VizroAI.dashboard` to generate Vizro dashboard.
+
+??? note "Note: API key"
+
+    Make sure you have followed the [LLM setup guide](../user-guides/install.md#set-up-access-to-a-large-language-model) and thatyour api key is set up in a `.env` file in the same folder as your Notebook file (`.ipynb`).
+
+## Run Vizro-AI dashboard
+
+### 1. Prepare the data and user prompt
+```py
+import vizro.plotly.express as px
+
+df = px.data.tips()
+
+user_question = """
+Create a one-page dashboard layout with the following components:
+
+1. Card:
+   - Position: Left of the page
+   - Size: Takes up 1/4 of the total page width
+   - Content: Display the text "This is Tips dataset"
+
+2. Table:
+   - Position: Right of the card
+   - Size: Takes up the remaining 3/4 of the page width
+   - Content: Display the Tips dataset
+"""
+```
+
+### 2. Generate and launch the dashboard
```

Comment:
Maybe we could try to avoid to repeat the prompt here? Then it is less text that is rewritten?

I always prefer on comprehensive example that does it all